compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -371,6 +371,7 @@ class OMR_EXTENSIBLE CodeGenerator
    TR_HasRandomGenerator randomizer;
 
    virtual bool supportsAtomicAdd() {return false;}
+   virtual bool supportsTM()    {return false;}
```

Comment:
In general, because CodeGenerator is an extensible class, marking a member function as virtual isn't what you should use unless you really need to achieve dynamic polymorphism.  In your case, you shouldn't use it.